modules/images/webp-uploads/rest-api.php

```diff
@@ -23,20 +23,30 @@ function webp_uploads_update_rest_attachment( WP_REST_Response $response, WP_Pos
 	}
 
 	$metadata = wp_get_attachment_metadata( $post->ID );
-	foreach ( $data['media_details']['sizes'] as $size => $details ) {
+	foreach ( $data['media_details']['sizes'] as $size => &$details ) {
 		if ( empty( $metadata['sizes'][ $size ]['sources'] ) || ! is_array( $metadata['sizes'][ $size ]['sources'] ) ) {
```

Comment:
See above, this could then become:
```suggestion
		if ( empty( $details['sources'] ) || ! is_array( $details['sources'] ) ) {
```

Reply:
Thank you. This feedback is addressed.